syscall/unistd.c

```diff
@@ -145,6 +144,34 @@ int oe_nanosleep(struct oe_timespec* req, struct oe_timespec* rem)
     return ret;
 }
 
+int oe_clock_gettime(int clock_id, struct oe_timespec* cur_time)
+{
+    int ret = 0;
+    oe_syscall_clock_gettime_ocall(&ret, clock_id, cur_time);
+    return ret;
+}
+
+int oe_gettimeofday(struct oe_timeval* tv, struct oe_timezone* tz)
+{
+    int ret = -1;
```

Comment:
coding style suggestion: It would be great to have space between control blocks. Please refer to the original `_syscall_gettimeofday` for the reference.